src/utils/diagram/floating-edge-util.tsx

```diff
@@ -0,0 +1,69 @@
+import { Position } from "react-flow-renderer/nocss";
```

Comment:
Because this isn't rendering any react, its extension should be `.ts` instead of `.tsx`.
You should add some tests of it. That would help explain what it does, and keep the coverage up.